packages/core/admin/ee/server/migrations/review-workflows.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * Determine if a content type has the review workflows feature enabled
+ * @param {Object} contentType
+ * @returns
+ */
+const hasRWEnabled = (contentType) => contentType?.options?.reviewWorkflows || false;
+
+/**
+ * Remove all stage information for all content types that have had review workflows disabled
+ */
+/* eslint-disable no-continue */
+const disableReviewWorkFlows = async ({ oldContentTypes, contentTypes }) => {
+  const uidsToRemove = [];
+  for (const uid in contentTypes) {
+    if (!oldContentTypes || !oldContentTypes[uid]) {
+      continue;
+    }
+
+    const oldContentType = oldContentTypes[uid];
+    const contentType = contentTypes[uid];
```

Comment:
just a minor edge case, what if you delete all content types? this would throw an error as `contentTypes` would be undefined.
```suggestion
    const contentType = contentTypes?.[uid];
```